frontend/src/pages/booking/components/BookingSummarySection/BookingSummarySection.tsx

```diff
@@ -0,0 +1,51 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import timeIcon from '../../../../common/assets/images/timeIcon.svg';
+import Button from '../../../../common/components/Button/Button';
+import TextWithIcon from '../../../../common/components/TextWithIcon/TextWithIcon';
+
+function BookingSummarySection() {
+  const handleBookingButtonClick = () => {};
+
+  return (
+    <StyledContainer>
+      <StyledWrapper>
+        <TextWithIcon iconSrc={timeIcon} iconName="시간 아이콘" text="15분" />
+        <StyledPrice>5,000원</StyledPrice>
+      </StyledWrapper>
+      <Button
+        customStyle={css`
+          flex-grow: 1;
+          padding: 0.8rem 0rem;
+        `}
+        onClick={handleBookingButtonClick}
+      >
+        예약하기
+      </Button>
```

Comment:
p4: 이 부분이 BookingButton으로 분리되고 children으로 들어오는 건 어떨까? 🤔 
Button이 밖에서 children으로 오면 BookingForm 컴포넌트에서 바로 데이터와 핸들러 함수를 넘겨줄 수 있을 거 같아서 말해봤어 ㅎㅎ 

```suggestion
<BookingSummarySection >
  <Button
        customStyle={css`
          flex-grow: 1;
          padding: 0.8rem 0rem;
        `}
        onClick={handleBookingButtonClick}
      >
        예약하기
      </Button>
</BookingSummarySection >
      
```

Reply:
이 컴포넌트는 예약 요약(시간.가격)과 버튼을 묶어서 보여주는 역할을 하는 컴포넌트로 만들었어. 
버튼 자체는 바뀔 가능성이 거의 없다고 판단해서 지금과 같은 고정된 구조를 가졌고, 추후에 외부에서 바꿀 일이 생긴다면 그때 분리해도 괜찮을 거 같아!

다만 버튼의 동작은 외부(BookingForm) 쪽의 상태가 필요하니 핸들러는 props로 주입받는 게 적절한거 같아👍👍

구조가 단순하고 props 깊이도 얕은 현재 상황에선 버튼을 고정된 구조로 관리하는 것도 좋을 거 같은데 로건은 어떻게 생각해??

Reply:
바뀔 가능성이 없어서 단순하게 만드는 것도 좋은 것 같네 ㅎㅎ 

---

버튼의 동작에 외부(BookingForm) 쪽의 상태가 필요해서, 이 Button이 BookingForm의 children이 되면 props로 주입 받기 보다는 바로 전달할 수 있을 거 같아서 말해봤었어!

그런데 데이지 말대로 구조가 단순하고 props 깊이도 얕으니까 고정된 구조로 관리해도 의존성이 명확하게 보여서 좋을 것 같네 👍
의견 좋은데요~~👍